pkgs/tools/graphics/wkhtmltopdf-bin/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchurl, wkhtmltopdf, xar, cpio }:
+
+stdenv.mkDerivation rec {
+
+  pname = "wkhtmltopdf-bin";
+  version = "0.12.6-1";
```

Comment:
It looks like [0.12.6-2](https://github.com/wkhtmltopdf/packaging/releases/tag/0.12.6-2) was a macOS-specific release and it doesn't contain binaries for Linux. `0.12.6-2` appears to only make the build usable on macOS version < 10.14 (https://github.com/wkhtmltopdf/wkhtmltopdf/issues/4716). Otherwise, I'd prefer to use `0.12.6-1` since it contains binaries for Linux and macOS 10.13 has reached its end-of-life.

Is there another reason for that release that you had in mind?

Reply:
ah, ok, strange that they did that.

`0.12.6-1` is fine then

Reply:
happy to stay with `0.12.6-1` but out of curiosity I downloaded `0.12.6-2` on maxOS 12.2.1 and it worked, so not sure what they tried to fix with build

Reply:
@nbr  sorry i didn't read you comment properly, the fix was for older macOS versions